domehunter/dome_control.py

```diff
@@ -0,0 +1,674 @@
+"""Run dome control on a raspberry pi GPIO."""
+
+
+import sys
+import time
+import warnings
+
+from gpiozero import Device, DigitalInputDevice, DigitalOutputDevice
+from gpiozero.pins.mock import MockFactory
+
+from ._astropy_init import *
+
+# if we want to use the automation hat status lights we need to
+# import the pimoroni led driver
+try:  # pragma: no cover
+    import sn3218
+    sn3218.disable()
+except OSError:  # pragma: no cover
+    warnings.warn(
+        "AutomationHAT hardware not detected, testing=True and"
+        " debug_lights=False recommended.")
+except:  # pragma: no cover
```

Comment:
```suggestion
except Exception:  # pragma: no cover
```
Will catch all other Exceptions. Shouldn't have a bare `except`.